examples/conditions/pull_approve.js

```diff
@@ -333,17 +333,32 @@ function reviewed({
       if (satisfied) {
         file.designatedReviewers.push(
           ..._.map(team, (unused, key) => ({builtin: 'fulfilled', scope})));
-        if (unrequest) unrequestedTeams.push(..._.keys(team));
-      } else {
-        requiredTeams.push(..._.keys(team));
-        if (request && !_.some(file.revisions, 'reviewed')) requestedTeams.push(..._.keys(team));
       }
     } else {
       file.designatedReviewers.push({builtin: 'anyone', omitBaseChanges});
       if (satisfied) file.designatedReviewers.push({builtin: 'fulfilled'});
     }
   }
 
+  if (team) {
+    if (allSatisfied) {
+      if (unrequest) unrequestedTeams.push(..._.keys(team));
+    } else {
+      requiredTeams.push(..._.keys(team));
```

Comment:
I think I actually prefer to always require/keep the teams' review until all the files are satisfied, maybe that could be a config in the reviewed like `unrequest: "participating" | "satisfied" | "never" = "participating";` instead of plain `unrequest: boolean = true`
I need to double check with my team if that's something desirable. Making it configurable could be nice though.

While this would solve the "rerequesting" issue with teams, it feels like a workaround, not a real fix